operator/operator_test.go

```diff
@@ -136,7 +141,9 @@ func initEnvironment(t *testing.T) *environment {
 		log.Fatal(err)
 	}
 
-	operator := Setup(cfg)
+	enclave := createRemoteEnclave(cfg)
+	operator := SetupWithEnclave(cfg, enclave)
```

Comment:
Can we instead parameterize the test so that it can be run with a remote as well as a prototype enclave.
This test here should test the operator and not be a remote enclave test, so I think it would be better to have this test still use the prototype.

Reply:
Added parameterization options for prototype enclave, remote enclave (dial), and remote enclave (wrapped around local prototype enclave).